packages/webpack-config/webpack.base.js

```diff
@@ -9,6 +9,15 @@ require('dotenv').config({ path: './.env' });
 const shell = require("shelljs");
 const fs = require("fs");
 
+const meta = {
+  title: "GC Digital Talent | Talent numérique du GC",
+  description: "Recruitment platform for digital jobs in the Government of Canada. Recrutement pour les emplois numériques au gouvernement du Canada.",
```

Comment:
The French version of _Recruitment platform_ does not include the word platform.

Reply:
Oops, that is an important piece